src/views/CreateDandisetView/CreateDandisetView.vue

```diff
@@ -3,31 +3,59 @@
     <v-card-title>
       <span class="text-h5">Register a new dataset</span>
     </v-card-title>
-    <v-card-text>
-      <v-text-field
-        v-model="name"
-        label="Name*"
-        hint="Provide a title for this dataset"
-        persistent-hint
-        :counter="120"
-        required
-      />
-      <v-textarea
-        v-model="description"
-        label="Description*"
-        hint="Provide a description for this dataset"
-        :counter="3000"
-        persistent-hint
-        required
-      />
-      <small>*indicates required field</small>
+    <v-card-text class="my-3">
+      <v-form>
+        <v-text-field
+          v-model="name"
+          label="Name*"
+          hint="Provide a title for this dataset"
+          persistent-hint
+          :counter="120"
+          required
+          outlined
+        />
+        <v-textarea
+          v-model="description"
+          label="Description*"
+          hint="Provide a description for this dataset"
+          :counter="3000"
+          persistent-hint
+          required
+          outlined
+          class="my-4"
+        />
+        <v-checkbox
+          v-model="embargoed"
+          hide-details
+          class="shrink mr-2 mt-0"
+        >
+          <template #label>
+            Embargo this dataset?
```

Comment:
We need a blurb here, perhaps in a mouseover tooltip. Casual users need enough information to understand that they should not check this unless they have an NIH grant.
We should also clarify that any award number can be specified at this time if there are multiple, and that you can change your answer later.

@waxlamp any suggestions?

Reply:
Did I not ever respond to this? I could have sworn I wrote out a full answer...

Here's a first attempt at a blurb:

```
Embargoed datasets are hidden from public access until a specific time period has elapsed. Uploading data to the DANDI archive under embargo requires a relevant NIH award number, and the data will be automatically published when the embargo period expires.
```